### PR TITLE
Prefer SSL connection to Mongo sources

### PR DESCRIPTION
**Pull request by louis-pie**
## Context

[AP-1693](https://transferwise.atlassian.net/browse/AP-1693) : Prefer SSL connection to Mongo sources
Make changes to PipelineWise, as well as local-dev and testing to prefer SSL connections to Mongo.
If SSL connection fails, a regular connection will be attempted.

## Types of changes

What types of changes does your code introduce to PipelineWise?
_Put an `x` in the boxes that apply_

- [x] New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] Description above provides context of the change
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Unit tests for changes (not needed for documentation changes)
- [x] CI checks pass with my changes
- [x] Bumping version in `setup.py` is an individual PR and not mixed with feature or bugfix PRs
- [x] Commit message/PR title starts with `[AP-NNNN]` (if applicable. AP-NNNN = JIRA ID)
- [x] Branch name starts with `AP-NNN` (if applicable. AP-NNN = JIRA ID)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions

[AP-1693]: https://transferwise.atlassian.net/browse/AP-1693?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -6,8 +6,8 @@
     LONG_DESCRIPTION = f.read()
 
 setup(name='pipelinewise',
-      python_requires='>=3.7,<3.10',
-      version='0.60.2b1',
+      python_requires='3.8.*',
```
Comment (amofakhar) [inline setup.py:9]:
```suggestion
      python_requires=='3.8.*',
```

File: setup.py [hunk 2]
```diff
@@ -6,8 +6,8 @@
     LONG_DESCRIPTION = f.read()
 
 setup(name='pipelinewise',
-      python_requires='>=3.7,<3.10',
-      version='0.60.2b1',
+      python_requires=='3.8.*',
```
Comment (amofakhar) [inline setup.py:9]:
```suggestion
      python_requires='==3.8.*',
```

**Review by teaf-wise — APPROVED** [review]

**Comment by louis-pie** [comment]
wise-bot emergency-merge GitHub Actions pretzel problem

**Pull request merged**